scripts/update/Updater.php

```diff
@@ -560,6 +561,11 @@ public function update($initialVersion)
         }
 
         $this->skip('4.3.0', '4.4.0');
+
+        if ($this->isVersion('4.4.0')) {
+            $this->getServiceManager()->register(TestCenterService::SERVICE_ID, new TestCenterService([]));
```

Comment:
Please remove the empty array constructor parameter for the `TestCenterService`.